populate_dev_data_handler.go

```diff
@@ -86,11 +86,14 @@ func populateDevData(w http.ResponseWriter, r *http.Request) {
 		"safari",
 	} {
 		// TODO(lukebjerring): Move wpt.fyi base URL to constant.
-		jsonURL := "https://wpt.fyi/json?platform=" + browserName
+		jsonURL := "https://wpt.fyi/results?platform=" + browserName
 		resp, err := client.Head(jsonURL)
 
-		if urlError, ok := err.(*url.Error); !ok || urlError.Err != errUseLastResponse {
-			fmt.Fprintf(w, "Failed to fetch latest run for %s: %s\n", browserName, urlError.Error())
+		if err == nil {
+			fmt.Fprintf(w, "Got unexpected non-redirect status %d for %s\n", resp.StatusCode, jsonURL)
+			continue
+		} else if urlError, ok := err.(*url.Error); !ok || urlError.Err != errUseLastResponse {
```

Comment:
The `err.(*url.Error)` construct scrambles my brain every time

Reply:
Yeah casting isn't so intuitive written this way.